settings.gradle

```diff
@@ -2,6 +2,13 @@ plugins {
     id 'de.fayard.refreshVersions' version '0.60.3'
 }
 
+dependencyResolutionManagement {
```

Comment:
This is the newer / correct way to do this in AGP 8, since we're referring to it in our README we may as well do it ourselves.